### PR TITLE
deploy: deployment utilities and cmd fixes

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

**Comment by mfojtik** [comment]
@kargakis i was thinking doing the same while working on cancel ;-)

**Comment by mfojtik** [comment]
[test]

File: pkg/cmd/cli/cmd/deploy.go [hunk 1]
```diff
@@ -279,17 +276,17 @@ func (o DeployOptions) retry(config *deployapi.DeploymentConfig) error {
 	deployment, err := o.kubeClient.ReplicationControllers(config.Namespace).Get(deploymentName)
 	if err != nil {
 		if kerrors.IsNotFound(err) {
-			return fmt.Errorf("unable to find the latest deployment (#%d).\nYou can start a new deployment using the --latest option.", config.Status.LatestVersion)
+			return fmt.Errorf("unable to find the latest deployment (#%d).\nYou can start a new deployment with 'oc deploy --latest dc/%s'.", config.Status.LatestVersion, config.Name)
```
Comment (mfojtik) [inline pkg/cmd/cli/cmd/deploy.go]: add todo to change this when we get `oc rollout latest`

**Comment by mfojtik** [comment]
LGTM (post-1.3.0)

File: pkg/cmd/cli/cmd/deploy.go (hunk 1 above)
Comment (kargakis) [inline pkg/cmd/cli/cmd/deploy.go]: I will rebase my PR with `oc rollout latest` on top of this anyway.

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**